medusa/server/web/home/handler.py

```diff
@@ -1571,6 +1571,14 @@ def editShow(self, show=None, location=None, allowed_qualities=None, preferred_q
                     logger.log("Unable to refresh show '{show}': {error}".format
                                (show=show_obj.name, error=e.message), logger.WARNING)
 
+            # Check if we should erase parsed cached results f or that show
+            do_erase_parsed_cache = False
+            for item in ['scene', 'anime', 'sports', 'air_by_date', 'dvd_order']:
+                if getattr(show_obj, item) != ast.parse(item):
```

Comment:
@p0psicles this checks

 if show_obj.scene != scene

So if user change any those vars, i set tje flag to clean